build_and_deploy/utils/workspace_artifacts_getter.ts

```diff
@@ -0,0 +1,307 @@
+import { ArtifactClient } from '../clients/artifacts_client';
+import * as deployUtils from './deploy_utils';
+import * as httpClient from 'typed-rest-client/HttpClient';
+import * as httpInterfaces from 'typed-rest-client/Interfaces';
+import { checkIfArtifactExists, checkIfNameExists, Resource } from './arm_template_utils';
+import { Artifact } from './artifacts_enum';
+import {SystemLogger} from "./logger";
+
+const userAgent: string = 'synapse-github-cicd-deploy-task'
+const requestOptions: httpInterfaces.IRequestOptions = {};
+const client: httpClient.HttpClient = new httpClient.HttpClient(userAgent, undefined, requestOptions);
+
+const artifactTypesToQuery:Artifact[] = [
```

Comment:
Is this list not exists in another file? pls cross check try reuse if already exists

Reply:
We have an enum which is used here. We don't have a list